sources/Interop/Windows/Windows/um/minidumpapiset/THREAD_WRITE_FLAGS.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/xmllite.h in the Windows SDK for Windows 10.0.26100.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop.Windows;
+
+/// <include file='THREAD_WRITE_FLAGS.xml' path='doc/member[@name="THREAD_WRITE_FLAGS"]/*' />
+public enum THREAD_WRITE_FLAGS
```

Comment:
Looks like this and a couple others should have `--with-attribute` entries:
```
--with-attribute
THREAD_WRITE_FLAGS=Flags
...=Flags
```

for any of the enums that can be flags